Bin/Snapshot.php

```diff
@@ -264,6 +291,10 @@ function () use ($tags, $newTag, $repositoryRoot, &$changelog) {
                 } else {
                     array_unshift($tags, 'HEAD');
 
+                    $changelogThemesRegex = sprintf(
+                        '/(%s)\(([^)]+)\)(.*)/m',
```

Comment:
```diff
- '/(%s)\(([^)]+)\)(.*)/m
+ '/^(%s)\(([^)]+)\)(.*)$/
```

I've added anchors, and removed the `m` option. Why do we need it?

Reply:
I would also recommend to name the groups (with `(?<x>)`) to get a more readable code in your loop bellow.

Reply:
Yep, the `m` option is totally useless here... I've added the named groups.